src/compat/e-132/ArkoudaRandomCompat.chpl

```diff
@@ -32,7 +32,70 @@ module ArkoudaRandomCompat {
     }
 
     proc ref sample(d: domain(?), n: int, withReplacement = false): [] d.idxType throws  where is1DRectangularDomain(d) {
-      return r.choice(d, n, withReplacement);
+      return choiceUniform(r, d, n, withReplacement);
+
+      /* _choice branch for uniform distribution */
+      proc choiceUniform(stream, X: domain(?), size: ?sizeType, replace: bool) throws
```

Comment:
Why doesn't `stream` need an intent?

Reply:
by leaving off a type intent, it's generic. So technically anything could be passed in as the `stream` variable

Since it's only ever called by line 35, we could give it the intent `: shared PCGRandomStream(eltType)`

The reason I didn't was because it didn't have a type intent in the chpl sorce code for the `1.32` release, so I wanted to make minimal modifications to fix the bug

https://github.com/chapel-lang/chapel/blob/1.32.0/modules/standard/Random.chpl#L328